### PR TITLE
Enhanced Placement of Icons in Select Captcha Type Options

### PR DESCRIPTION
**Pull request by shervElmi**
This PR aims to improve the placement of icons within the Select Captcha Type options. The proposed changes are part of the Interface Reviews conducted by Razvan.

Figma URL: [Link to Figma Design](https://www.figma.com/file/7Kwlv3H2ShYEI0wekWnPBb/Interface-Review-Fixes?type=design&node-id=702-122&t=DsmaDrn9AMRMkzcv-4)

## QA URL:
https://qa.formidableforms.com/sherv2/wp-admin/admin.php?page=formidable-settings

## Testing Instructions:
1. Go to `WP Admin > Formidable > Global Settings > Captcha` page.
2. Verify that the Select Captcha Type option displays the icon as shown in the "After" screenshot below.

### Before
<kbd>
image
</kbd>

### After
<kbd>
image
</kbd>

### CODE REVIEW

**Review by garretlaxton — APPROVED** [review]
This looks great!

File: classes/views/frm-settings/captcha/captcha.php [hunk 1]
```diff
@@ -11,13 +11,13 @@
 <div class="frm_captchas frm-long-icon-buttons">
 	<input type="radio" name="frm_active_captcha" id="frm-recaptcha" value="recaptcha" data-frmhide="#hcaptcha_settings" data-frmshow="#recaptcha_settings" <?php checked( $frm_settings->active_captcha, 'recaptcha' ); ?> />
 	<label for="frm-recaptcha">
-		<?php echo esc_html_e( 'reCAPTCHA', 'formidable' ); ?>
 		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 38"><path fill="#1C3AA9" d="M38.6 19a19.8 19.8 0 0 0 0-.7V2.8L34.3 7a19.1 19.1 0 0 0-30.1.6l7 7a9.3 9.3 0 0 1 2.9-3.1c1.2-1 3-1.8 5.3-1.8l.7.1a9.3 9.3 0 0 1 7.1 4.3l-5 5h16.4"/><path fill="#4285F4" d="M19.3 0H3l4.3 4.3a19.1 19.1 0 0 0 .5 30l7.1-7a9.3 9.3 0 0 1-3.2-2.8c-1-1.2-1.7-3-1.7-5.4v-.6a9.3 9.3 0 0 1 4.4-7.1l5 5V0"/><path fill="#ABABAB" d="M.3 19.1v16.4l4.3-4.3a19.1 19.1 0 0 0 30-.5l-7-7.2a9.3 9.3 0 0 1-2.8 3.2c-1.3 1-3 1.8-5.4 1.8a2 2 0 0 1-.7-.1 9.3 9.3 0 0 1-7-4.3l4.9-5H.3"/></svg>
+		<?php echo esc_html_e( 'reCAPTCHA', 'formidable' ); ?>
 	</label>
```
Comment (omidakhavans) [inline classes/views/frm-settings/captcha/captcha.php:16]: @shervElmi  `esc_html_e` doesn't need `echo`

Comment (shervElmi) [reply classes/views/frm-settings/captcha/captcha.php:16]: Yes, you're correct, @omidakhavans. I just moved this line. Thanks for your careful eyes.

**Review by Crabcyborg — APPROVED** [review]
Looks good! Thanks team! ❤️

**Comment by shervElmi** [comment]
> Looks good! Thanks team! ❤️

Thanks so much, Mike!

**Pull request merged**